build-managed.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 [platform] [useservergc]"
+    echo
+    echo "platform can be: FreeBSD, Linux, NetBSD, OSX, Windows."
+    echo "useservergc - Switch used by configure the hosted coreclr instance when executing tests."
+    echo
+}
+
+
+__scriptpath=$(cd "$(dirname "$0")"; pwd -P)
+
+__UnprocessedBuildArgs=
+__TestNugetRuntimeId=-distroRid
+__BuildOS=-os
+__TargetOS=-target-os
+__ServerGC=0
+
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    case $lowerI in
```

Comment:
I also want it to be in dir.props. There is a section where we are already defining `TestNugetRuntimeId` but as @joperezr pointed out, when calling this variables from the scripts they overwrite all the places were we have them. Just putting them in dir.props is not the same behavior.
I haven't test it to see if it would be a breaking change. But for a next iteration we could do it.